projects/valtimo/components/src/lib/components/left-sidebar/left-sidebar.component.ts

```diff
@@ -149,11 +150,12 @@ export class LeftSidebarComponent implements OnInit, AfterViewInit, OnDestroy {
     this.bodyStyle.cursor = 'auto';
     this.bodyStyle.userSelect = 'auto';
     this.isResizing$.next(false);
+    localStorage.setItem('menuWidth', this.resizedWidth.toString());
```

Comment:
I don't think you need the this.resizedWith class property, here you can just use `localStorage.setItem('menuWidth', this.menuWidth$.getValue().toString());`